org.eclipse.m2e.pde.feature/feature.xml

```diff
@@ -1,63 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
-      id="org.eclipse.m2e.pde.feature"
-      label="%featureName"
-      version="1.20.0.qualifier"
-      provider-name="%providerName"
-      plugin="org.eclipse.m2e.core"
+      version="1.20.100.qualifier"
```

Comment:
Some important attributes and other content have vanished.

Reply:
Sorry, fixed and force pushed to update this PR.